### PR TITLE
Check for event loop and nsys report in any profiling workflow

### PR DESCRIPTION
**Pull request by gartung**

### CODE REVIEW

**Comment by gartung** [comment]
@smuzaffar

**Comment by aandvalenzuela** [comment]
Hello @gartung, is there a way to test these changes? I guess we need to get this in for running it, right?

**Comment by gartung** [comment]
It is run as part of the integration build. The standard IB should tell if there are syntax errors.  It changes what gets linked on the IB dashboard.

**Comment by gartung** [comment]
please test

File: report-summary-merged-prs [hunk 1]
```diff
@@ -1467,8 +1467,8 @@ if __name__ == "__main__":
     MAGIC_COMMAND_FIND_IGPROF = 'test -d ' + JENKINS_ARTIFACTS_DIR + '/igprof/RELEASE_NAME'
     MAGIC_COMMAND_FIND_PROFILING = 'test -d ' + JENKINS_ARTIFACTS_DIR + '/profiling/RELEASE_NAME'
     MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER1 = 'ls '+JENKINS_ARTIFACTS_DIR+'/profiling/RELEASE_NAME/ARCHITECTURE/WORKFLOW/step3_*.resources.json 2>/dev/null | head -1 | sed "s|.*/RELEASE_NAME/||;s|.json$||"'
-    MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER2 = 'test -s ' + JENKINS_ARTIFACTS_DIR + '/igprof/RELEASE_NAME/ARCHITECTURE/profiling/WORKFLOW/sorted_RES_CPU_step3.txt && echo ARCHITECTURE/profiling/WORKFLOW/sorted_RES_CPU_step3.txt'
-    MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER3 = 'test -s ' + JENKINS_ARTIFACTS_DIR + '/profiling/RELEASE_NAME/ARCHITECTURE/WORKFLOW/step3_gpu_nsys.txt && echo ARCHITECTURE/WORKFLOW/step3_gpu_nsys.txt'
+    MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER2 = 'ls ' + JENKINS_ARTIFACTS_DIR + '/igprof/RELEASE_NAME/ARCHITECTURE/profiling/*/sorted_RES_CPU_step3.txt 2>/dev/null | head -1'
+    MAGIC_COMMAND_FIND_PROFILING_CHECKS_FILTER3 = 'ls ' + JENKINS_ARTIFACTS_DIR + '/profiling/RELEASE_NAME/ARCHITECTURE/*/step3_gpu_nsys.txt 2>/dev/null | head -1'
```
Comment (smuzaffar) [inline report-summary-merged-prs:1471]: @gartung , previously these commands were returning `ARCHITECTURE/profiling/WORKFLOW/sorted_RES_CPU_step3.txt` and now it return full path which will break the access url. May be better strip off the unwanted path e.g.
```
'ls ' + JENKINS_ARTIFACTS_DIR + '/igprof/RELEASE_NAME/ARCHITECTURE/profiling/*/sorted_RES_CPU_step3.txt | sed 's|.*/igprof//|RELEASE_NAME|' | head -1
```
and somehting simillar for `/profiling/RELEASE_NAME`

```

Comment (gartung) [reply report-summary-merged-prs]: Updated

**Comment by smuzaffar** [comment]
thanks @gartung , I am merging it now

**Pull request merged**